cmd/resources/resources.go

```diff
@@ -117,39 +138,42 @@ func GetTemplateData(fileName string) (TemplateData, error) {
 }
 
 func getCmdUse(method string, op *openapi3.Operation, spec *openapi3.T) string {
-	methodMap := map[string]string{
-		"GET":    "get",
-		"POST":   "create",
-		"PUT":    "replace", // TODO: confirm this
-		"DELETE": "delete",
-		"PATCH":  "update",
-	}
-
-	use := methodMap[method]
-
-	var schema *openapi3.SchemaRef
-	for respType, respInfo := range op.Responses.Map() {
-		respCode, _ := strconv.Atoi(respType)
-		if respCode < 300 {
-			for _, s := range respInfo.Value.Content {
-				schemaName := strings.TrimPrefix(s.Schema.Ref, "#/components/schemas/")
-				schema = spec.Components.Schemas[schemaName]
-			}
-		}
-	}
-
-	if schema == nil {
-		// probably won't need to keep this logging in but leaving it for debugging purposes
-		log.Printf("No response type defined for %s", op.OperationID)
-	} else {
-		for propName := range schema.Value.Properties {
-			if propName == "items" {
-				use = "list"
-				break
-			}
-		}
-	}
-	return use
+	return strcase.ToKebab(op.OperationID)
+
+	// TODO: work with operation ID & response type to stripe out resource name and update post -> create, get -> list, etc.
```

Comment:
see comment in main description - going to bring this back in but for now, to remove duplicate subcommands, using just the OperationId